### PR TITLE
BEST4 enterocyte and subtypes

### PR DESCRIPTION
**Pull request by JABelfiore**
class hierarchy revision and 2 subclasses added

### CODE REVIEW

**Comment by dosumis** [comment]
Looks good, but better to make part relationships to epithelium of colon/small intestine.

Also, there is some potential material on function in source papers that could go in extended description.

**Comment by anitacaron** [comment]
Please always provide a descriptive PR title to clarify the changes in the release notes. 😄

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -30811,12 +30813,12 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044") rdfs:comment
...
-# Class: obo:CL_4030026 (BEST4+ intestinal epithelial cell, human)
+# Class: obo:CL_4030026 (BEST4+ enterocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An absorptive epithelial cell of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An enterocyte of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
 AnnotationAssertion(terms:date obo:CL_4030026 "2022-09-07T12:34:20Z"^^xsd:dateTime)
-AnnotationAssertion(rdfs:label obo:CL_4030026 "BEST4+ intestinal epithelial cell, human")
-SubClassOf(obo:CL_4030026 obo:CL_0000677)
+AnnotationAssertion(rdfs:label obo:CL_4030026 "BEST4+ enterocyte")
+SubClassOf(obo:CL_4030026 obo:CL_0000584)
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:30821]: consider to add the following logical definition to the cells expressing BEST4+
`and expresses some bestrophin-4`

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32234,6 +32236,26 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:34497389") obo:IAO_0000115 obo:CL_4047001 "An enterocyte of the human small intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
+AnnotationAssertion(terms:contributor obo:CL_4047001 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047001 "2024-07-25T13:05:50Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4047001 "small intestine BEST4+ enterocyte")
+SubClassOf(obo:CL_4047001 obo:CL_4030026)
+SubClassOf(obo:CL_4047001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002108))
+SubClassOf(obo:CL_4047001 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_Q8NFU0))
+
+# Class: obo:CL_4047002 (BEST4+ colonocyte)
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:32249]: consider to add the related synonym
has related synonyms
BEST4 mature colonocyte https://elifesciences.org/reviewed-preprints/88879#

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -30811,12 +30813,12 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044") rdfs:comment
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
-# Class: obo:CL_4030026 (BEST4+ intestinal epithelial cell, human)
+# Class: obo:CL_4030026 (BEST4+ enterocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An absorptive epithelial cell of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An enterocyte of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
```
Comment (AvolaAmg) [inline src/ontology/cl-edit.owl:30818]: consider to add the broad synonym
BEST4+ epithelial cell https://www.nature.com/articles/s41586-021-03852-1
BEST4+ cell
and
The related synonym
BEST4-expressing enterocyte (same paper)

**Review by AvolaAmg — COMMENTED** [review]
Please see in line comments

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (dosumis) [reply src/ontology/cl-edit.owl:30818]: we decided to add as exact

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (Caroline-99) [reply src/ontology/cl-edit.owl:32249]: `BEST4+ mature colonocyte` should be a related synonym to the term `BEST4+ colonocyte `and not `BEST4+ enterocyte`

**Review by Caroline-99 — DISMISSED** [review]

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (Caroline-99) [reply src/ontology/cl-edit.owl:30818]: @JABelfiore Have these terms been added as exact synonyms? I can only see BEST4+ intestinal epithelial cell, human as exact synonym

**Comment by Caroline-99** [comment]
The QC is still failing. This seems to be because each of these classes (CL:4047001 and CL:4047002) is associated with multiple, conflicting definitions. I think @JABelfiore you might need to change the CL Ids manually.                  
  @aleixpuigb could you please check that? 
image

**Comment by aleixpuigb** [comment]
> The QC is still failing. This seems to be because each of these classes (CL:4047001 and CL:4047002) is associated with multiple, conflicting definitions. I think @JABelfiore you might need to change the CL Ids manually. @aleixpuigb could you please check that? image

That's correct, Those ID were used for cycling cells.

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
> The QC is still failing. This seems to be because each of these classes (CL:4047001 and CL:4047002) is associated with multiple, conflicting definitions. I think @JABelfiore you might need to change the CL Ids manually. @aleixpuigb could you please check that? image

@aleixpuigb it seems that all the cells that Jasmine added have been removed. I can't find them anymore on this branch. I think this might have happened when Jasmine was trying to resolve conflicts.

**Comment by dosumis** [comment]
#gogoeditdiff

**Review by dosumis — APPROVED** [review]

**Pull request merged**